libafl_frida/src/coverage_rt.rs

```diff
@@ -23,7 +33,18 @@ struct CoverageRuntimeInner {
 
 /// Frida binary-only coverage
 #[derive(Debug)]
-pub struct CoverageRuntime(Pin<Rc<RefCell<CoverageRuntimeInner>>>);
+pub struct CoverageRuntime {
+    inner: Pin<Rc<RefCell<CoverageRuntimeInner>>>,
+    inner_bbs: Pin<Rc<RefCell<CoverageRuntimeInner>>>,
+    /// The memory ranges of this target
+    ranges: RangeMap<usize, (u16, String)>,
+    save_dr_cov: bool,
+    coverage_directory: PathBuf,
+    basic_blocks: HashMap<u64, DrCovBasicBlock>,
+    cnt: usize,
+    max_cnt: usize,
```

Comment:
The name should clearly indicate it's dr_cov specific (or potentially both values should live in an extra DrCov struct)

Reply:
Agree. Make an extra DrCov struct please.